index.md

```diff
@@ -1,3 +1,6 @@
 ---
```

Comment:
# This is an <h1> header,larger one
## This is an<h2> header, h2 heading
### This is an <h6> header, h6 heading

Reply:
heading sizes